src/backend/src/modules/pia-intake/metadata/pia-status.metadata.ts

```diff
@@ -283,8 +283,63 @@ export const piaStatusMetadata: Partial<
           },
         ],
       },
+      [PiaIntakeStatusEnum.PENDING_COMPLETION]: {
+        allow: true,
+      },
+    },
+  },
+  [PiaIntakeStatusEnum.PENDING_COMPLETION]: {
+    updates: {
+      allow: false,
```

Comment:
Missing override to allow commenting. Correct me if I am wrong, but I think comments are allowed in Pending Completion.

Reply:
Comments are not allowed in the 'Pending completion' status, to my knowledge. The PIA is _effectively_ complete, it is simply pending upload to the PID by a CPO.

Reply:
yes, comments does not allow in pending and complete status, I can recall that

Reply:
Great, thanks 👍